include/capypdf.h.in

```diff
@@ -298,6 +298,14 @@ typedef enum {
     CAPY_FORM_ACTION_EMBEDFORM = (1 << 12),
 } CapyPDF_Form_Submit_Action_Flags;
 
+typedef enum {
+    CAPY_PAGE_LABEL_NUMBER_STYLE_DECIMAL,
+    CAPY_PAGE_LABEL_NUMBER_STYLE_ROMAN_UPPER,
+    CAPY_PAGE_LABEL_NUMBER_STYLE_ROMAN_LOWER,
+    CAPY_PAGE_LABEL_NUMBER_STYLE_LETTER_UPPER,
+    CAPY_PAGE_LABEL_NUMBER_STYLE_LETTER_LOWER
+} CapyPDF_PageLabelNumberStyle;
+
```

Comment:
Just a few days ago the naming was standardised so that enums are like `CapyPDF_Spaces_Between_Words` wheres structs are like `CapyPDF_NoSpacesBetweenWords`.